libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -211,13 +206,12 @@ private void upgradeTo9Dot2() {
     }
 
     // TODO: Remove upgrade step in Mobile SDK 12.0
-    private void upgradeTo10Dot1Dot1() {
+    private void upgradeFromVersions9_2_0Thru10_1_1To10_1_1PasscodeFixes() {
```

Comment:
These names are cumbersome, but I was getting really confused by which method was doing what.  I think this more accurately describes what is happening, but it is only a cosmetic change.  I will not protest reverting these method names.